File: Info/OIS Arena 2/arena2.pas
procedure capture(N, E: longint; var X, Y, points: array of longint);
begin
    (* insert your code here *)
    points[0] := 42;
end;


const
    MAXE = 1000000;
var
    N, E, i : longint;
    X, Y, P : array[0..MAXE-1] of longint;
    fr, fw  : text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N, E);
    for i:=0 to E-1 do
        readln(fr, X[i], Y[i]);

    capture(N, E, X, Y, P);
    for i:=0 to N-1 do
        write(fw, P[i], ' ');
    writeln(fw);
    close(fr);
    close(fw);
end.

File: Info/OIS Arena 2/arena2.cpp
#include <stdio.h>
#include <assert.h>

#define MAXE 1000000

void capture(int N, int E, int X[], int Y[], int points[]) {
    // insert your code here
    points[0] = 42;
}


int X[MAXE];
int Y[MAXE];
int P[MAXE];

int main() {
    FILE *fr, *fw;
    int N, E, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(2 == fscanf(fr, "%d %d", &N, &E));
    for(i=0; i<E; i++)
        assert(2 == fscanf(fr, "%d %d", &X[i], &Y[i]));
    
    capture(N, E, X, Y, P);
    for (i=0; i<N; i++)
        fprintf(fw, "%d ", P[i]);
    fprintf(fw, "\n");
    fclose(fr);
    fclose(fw);
    return 0;
}
